modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc

```diff
@@ -59,3 +59,39 @@ chmod +x ./1-prepare.sh ./2-migrate.sh ./3-subscribe.sh ./4-wait.sh
 
 * Open {prod-short} dashboard and log in.
 
+[IMPORTANT]
+====
+If migration failed, use the following steps to rollback to previous {prod-short} installation.
```

Comment:
If migration fails, use the following steps to roll back to the previous {prod-prev-short} installation.